### ISSUE TITLE
[Bug] - Not accessible with the native iOS screenreader (VoiceOver)

### ISSUE THREAD

**Issue by AnonymousAccessibilityAdvocate** [simplex-chat/simplex-chat#7274]
### Is there an existing issue for this?

- [x] I have searched the existing issues

### Platform

iOS

### OS version

iOS 26

### App version

All

### Current Behavior

VoiceOver cannot read chat threads. All elements (buttons) need labeling for screenreaders to read them.

### Expected Behavior

Users should be able to navigate the app with a screenreader. 

### Steps To Reproduce

Open app with VoiceOver enabled. 

### Relevant log output

```shell

```
